examples/guided.py

```diff
@@ -54,8 +65,14 @@ def ask_user_questions():
 	# 1. Check if it has partitions
 	# 3. Check that we support the current partitions
 	# 2. If so, ask if we should keep them or wipe everything
-	if archinstall.arguments['harddrive'] and archinstall.arguments['harddrive'].has_partitions():
-		archinstall.log(f"{archinstall.arguments['harddrive']} contains the following partitions:", fg='yellow')
+	if (
+		archinstall.arguments['harddrive']
+		and archinstall.arguments['harddrive'].has_partitions()
+	):
```

Comment:
Disclaimer (going to be a lot of these), this following is my personal opinion:

This decreases readability.